exercises/querying-workflows/solution/workflow.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+from typing import List
+
+from temporalio import workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+
+@workflow.defn
+class MyWorkflow:
+    def __init__(self) -> None:
+        self._pending_greetings: asyncio.Queue[str] = asyncio.Queue()
```

Comment:
Mentioned in signal course content, but I think you should just use a regular list or a deque or something. Don't need the asyncio collection.